### PR TITLE
New Cheat Sheet added for pdfimage

### PR DESCRIPTION
**Pull request by prasadram**
https://duck.co/ia/view/pdfimages_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@prasadram Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/pdfimage.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "id": "pdfimage_cheat_sheet", 
+    "name": "pdfimage",
+    "description": "Pdfimages saves images from a Portable Document Format (PDF) file as Portable Pixmap (PPM), Portable Bitmap (PBM), or JPEG files. ", 
+    "metadata": { 
+        "sourceName": "Linux ManualPage",
+        "sourceUrl": "http://www.linuxcommand.org/man_pages/pdfimages1.html "
+    },
+    "template_type" : "keyboard",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/pdfimage.json]: It looks like these are command line commands?

Lets use the `terminal` template

**Comment by moollaza** [comment]
@prasadram the source link suggests that the name is `pdfimage**s**` -- I think we should change the content/ID to reflect that?

**Comment by GuiltyDolphin** [comment]
@prasadram The `example query` should be `pdfimage cheat sheet`, not `pdfimage cheat cheet`.

File: share/goodie/cheat_sheets/json/pdfimage.json (hunk 1 above)
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/pdfimage.json]: I agree with @moollaza, take a look:

screenshot from 2016-02-08 16 26 40

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pdfimage cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/pdfimage_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/pdfimages.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+    "id": "pdfimages_cheat_sheet", 
+    "name": "pdfimages",
+    "description": "Pdfimages saves images from a Portable Document Format (PDF) file as Portable Pixmap (PPM), Portable Bitmap (PBM), or JPEG files. ", 
+    "metadata": { 
+        "sourceName": "Linux ManualPage",
+        "sourceUrl": "http://www.linuxcommand.org/man_pages/pdfimages1.html "
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/pdfimages.json]: Get rid of the space at the end of the URL.

File: share/goodie/cheat_sheets/json/pdfimages.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
+    "id": "pdfimages_cheat_sheet", 
+    "name": "pdfimages",
+    "description": "Pdfimages saves images from a Portable Document Format (PDF) file as Portable Pixmap (PPM), Portable Bitmap (PBM), or JPEG files. ", 
+    "metadata": { 
+        "sourceName": "Linux ManualPage",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/pdfimages.json]: Lets use `LinuxCommand`

File: share/goodie/cheat_sheets/json/pdfimages.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                "key": "-q",
+                "val": "Don’t print any messages or errors"
+            },
+            {
+                "key": "-v",
+                "val": "Print copyright and version information"
+            },
+            {
+                "key": "-h",
+                "val": "Print usage information  (-help and --help are equivalent)"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/pdfimages.json]: How about `(equivalent to -help and --help)`

File: share/goodie/cheat_sheets/json/pdfimages.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                "val": "Specifies the first page to scan"
+
+            },
+            {
+                "key": "-l number",
+                "val": "Specifies the last page to scan"
+            },
+            {
+                "key": "-j",
+                "val": "Normally, all images are written as PBM or  PPM files.With this option,images in DCT format are  saved as JPEG files.All non-DCT images are saved in PBM/PPM format as usual"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/pdfimages.json]: Just use `Save images in DCT format as JPEG` or similar.

File: share/goodie/cheat_sheets/json/pdfimages.json (hunk 3 above)
Comment (prasadram) [inline share/goodie/cheat_sheets/json/pdfimages.json]: @GuiltyDolphin the url is pointing to man pages so why can't we use Linux ManualPage?

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/pdfimages.json]: We typically use the domain (or main name) of the source. In this case that is `LinuxCommand`.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pdfimage cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/pdfimage_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/pdfimages.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
+{
+    "id": "pdfimages_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/pdfimages.json]: @prasadram this doesn't match your `id` -- should be `pdfimage_cheat_sheet` (no plural!)

Please update the filename to `pdfimage.json` and the id to `pdfimage_cheat_sheet`

**Comment by alohaas** [comment]
@prasadram, thanks again for creating the PDF Images! You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Linux.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to Linux. 

OR

- Test live Linux-related Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Pull request merged**